recipes/nbgrader/build.sh

```diff
@@ -0,0 +1,4 @@
+"${PYTHON}" setup.py install --single-version-externally-managed --record=record.txt
+
+#"${PREFIX}/bin/jupyter-nbextension" install --sys-prefix --overwrite --py nbgrader
```

Comment:
It can. But when it has multiple lines a separate file is probably better. I believe that my comment was more due to the fact that this line was commented and maybe, unnecessary.